orchestration/dags/jobs/ml/algo_training.py

```diff
@@ -211,15 +221,18 @@
     (
         start
         >> import_recommendation_data["training"]
-        >> import_recommendation_data["validation"]
-        >> import_recommendation_data["test"]
+        >> [
+            import_recommendation_data["validation"],
+            import_recommendation_data["test"],
+        ]
         >> gce_instance_start
         >> fetch_code
         >> install_dependencies
-        >> store_recommendation_data["training"]
-        >> store_recommendation_data["validation"]
-        >> store_recommendation_data["test"]
-        >> training
+        >> store_data["training"]
+        >> store_data["validation"]
+        >> train
+        >> store_data["test"]
```

Comment:
(?) Pourquoi est-ce que tu as mis ces store de données après le train ?

Reply:
On a pas besoin de store test avant le train donc je me suis dit que c'était plus logique mais on peut le mettre avant

Reply:
De manière générale c'est une bonne pratique de mettre les étapes courtes avant les étapes longues comme ça si ça plante tu peux le détecter plus vite et ne pas attendre une itération de train -> chaud pour le mettre avant

Reply:
Ok